test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -435,9 +435,9 @@ TEST(mongoUpdateContext_withOnchangeSubscriptions, Cond1_updateMatch)
     EXPECT_EQ(SccOk, ms);
 
     /* Check lastNotification */
```

Comment:
I'd do a different thing, to avoid excesive "issue-cracy" ;): I would add as a comment to #1476. #1476 will not be closed until the `-noCache` funcionality would be "in good shape".